README.md

```diff
@@ -0,0 +1,25 @@
+<p align="center">
+  <img style="max-width: 100%;" alt="Welcome to Create UI" src="readmeImages/projectLogo.png"/>
+</p>
+
+## Development
+
+This codebase comes with a development environment to facilitate enhancements and bug fixes. It allows developers to quickly iterate on the code and verify changes instantly.
+
+### Running the development environment
+
+From the root directory, run:
```

Comment:
It might seem kind of silly but on the off-chance the person reading this is new to NPM or JS development can we insert the step for installing dependencies here?

---

Install dependencies:

```
npm install
```

Reply:
https://github.com/vectara/react-search/pull/4/commits/ad9a0b57138d8f7c6269c15dc1e42377b95eae11